src/js/api/githubService.js

```diff
@@ -70,17 +84,7 @@ export function getAllPullRequests(repoNames) {
     pullRequests.sort((a, b) =>
       new Date(b.updated_at).getTime() - new Date(a.updated_at).getTime());
 
-    return Promise.map(pullRequests, pullRequest => {
```

Comment:
I added that in my comments PR, it was used to get the comment bodies and parse out emojis. It's possible I was double-fetching, so as long as it works, we ⚡ 💃